package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "postinstall": "node ./scripts/postinstall.js",
     "start": "cross-env NODE_ENV=development webpack-dev-server",
+    "mobile" : "cross-env NODE_ENV=development webpack-dev-server --useLocalIp --host 0.0.0.0",
```

Comment:
@brianpeiris is about to add [something similar](https://github.com/mozilla/mr-social-client/pull/64/files#diff-11e9f7f953edc64ba14b0cc350ae7b9d) in his PR. You can probably remove this.